sql/core/src/test/resources/sql-tests/inputs/postgreSQL/float4.sql

```diff
@@ -106,7 +106,6 @@ SELECT smallint(float('32767.6'));
 SELECT smallint(float('-32768.4'));
 SELECT smallint(float('-32768.6'));
 SELECT int(float('2147483520'));
-SELECT int(float('2147483647'));
```

Comment:
These tests are copied from pgsql and we shouldn't change it. We just need to re-generate the answer file and keep the actual result as it is.